packages/slice-machine/package.json

```diff
@@ -8,10 +8,6 @@
     "url": "ssh://git@github.com/prismicio/slice-machine.git",
     "directory": "packages/slice-machine"
   },
-  "_moduleAliases": {
```

Comment:
yeet

Reply:
moduleAliases has caused a lot of headaches, so it's nice to see it being phased out